src/Csv/CsvRuntime.fs

```diff
@@ -212,6 +212,14 @@ type CsvFile<'RowType> private (rowToStringArray:Func<'RowType,string[]>, dispos
   interface IDisposable with
     member __.Dispose() = disposer.Dispose()
 
+
+
+  /// [omit]
+  [<EditorBrowsableAttribute(EditorBrowsableState.Never)>]
+  [<CompilerMessageAttribute("This method is intended for use in generated code only.", 10001, IsHidden=true, IsError=false)>]
+  static member CreateEmpty (rowToStringArray, rows:seq<'RowType>, headers, numberOfColumns, separators, quote) =    
+    new CsvFile<'RowType>(rowToStringArray, { new IDisposable with member x.Dispose() =() }, rows, headers, numberOfColumns, separators, quote)
```

Comment:
Other direction, this if for writing to a string (Saving). stringArrayToRow is indeed dropped.